src/common/API/storageAPI.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
when you disable lint rules there should be a corresponding comment explaining why it is disabled - lint rules generally exist for a good reason

Reply:
ok will remove the esline disable